ros/src/computing/planning/motion/packages/waypoint_follower/nodes/mpc_follower/vehicle_model/vehicle_model_bicycle_kinematics.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mpc_follower/vehicle_model/vehicle_model_bicycle_kinematics.h"
+
+KinematicsBicycleModel::KinematicsBicycleModel(const double &wheelbase, const double &steer_lim_deg, const double &steer_tau)
+    : VehicleModelInterface(/* dim_x */ 3, /* dim_u */ 1, /* dim_y */ 2)
+{
+    wheelbase_ = wheelbase;
+    steer_lim_deg_ = steer_lim_deg;
+    steer_tau_ = steer_tau;
+};
+KinematicsBicycleModel::~KinematicsBicycleModel(){};
+
+void KinematicsBicycleModel::calculateDiscreteMatrix(Eigen::MatrixXd &Ad, Eigen::MatrixXd &Bd,
+                                                     Eigen::MatrixXd &Cd, Eigen::MatrixXd &Wd, const double &dt)
+{
+    auto sign = [](double x) { return (x > 0.0) - (x < 0.0); };
+    static const double DEG2RAD = M_PI / 180.0;
```

Comment:
The functionality to change angle units is already available in `amathutils`, please update

Reply:
remove definition, use function of amathutils